### PR TITLE
[pt-br] Add pt-br/docs/reference/glossary/istio.md

### PR DESCRIPTION
**Pull request by MrErlison**
Added translation for portuguese of the glossary term istio.

### CODE REVIEW

File: content/pt-br/docs/reference/glossary/istio.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+tags:
+- networking
+- architecture
+- extension
+---
+  Uma plataforma aberta (não específica do Kubernetes) que fornece uma maneira uniforme de integrar microsserviços, gerenciar o fluxo de tráfego, aplicar políticas e agregar dados de telemetria.
+
+<!--more--> 
+
+Adicionar o Istio não requer alterar o código da aplicação. É uma camada de infraestrutura entre um serviço e a rede, quando combinada com os serviços de implantações, é comumente referida como uma malha de serviço. A camada de gerenciamento do Istio abstrai a plataforma de gerenciamento do cluster principal, que pode ser Kubernetes, Mesosphere, etc.
```
Comment (stormqueen1990) [inline content/pt-br/docs/reference/glossary/istio.md:19]: "serviços de implantações" in this sentence is a bit far from the original sentence. Perhaps "instalações de serviços" would be a better fit here?

**Review by stormqueen1990 — COMMENTED** [review]
Hello there, @MrErlison!

Thank you very much for your contribution! 🎉 

I found one sentence that needs to be rethought. Other than that, LGTM.

**Comment by edsoncelio** [comment]
Thanks!
/approve

**Pull request merged**